lib/vagrant-service-manager/command.rb

```diff
@@ -76,29 +84,76 @@ def copy_from_box(hIP, hport, husername, hprivate_key_path, source, destination)
         end
       end
 
-      def execute_docker_info
-        # this execute the operations needed to print the docker env info
+      def check_if_a_service_is_running?(service)
+        command = "systemctl status #{service}"
```

Comment:
@navidshaikh We wont be able to check if k8s is setup and running just using systemctl status #{service}.  So just a info ahead of time. We have to tweak the code to run some other commands to check